assets/section-footer.css

```diff
@@ -271,7 +271,7 @@
   .localization-form:only-child {
     justify-content: start;
     width: auto;
-    margin: 0 0 0 -1rem;
+    margin: 0 0 0 var(--inputs-border-width);
```

Comment:
My idea for the borders overlapping would be to use the variable and add it as margin, but like @kmeleta  mentioned, maybe this can be tackled on a separate PR

https://github.com/Shopify/dawn/pull/1097/commits/e8a5780c64f9f9349f7c4a5c5fd4a211a8f97897

On an unrelated note, @melissaperreault I saw that the range the minimum for thickness is 0 where I am thinking it should be 1 so we can always have a border 🤔 

https://screenshot.click/06-15-hebi2-a3jwl.mp4

Reply:
Solving the border issue with margin may be ok for inputs considering the limited number of places where the issue may exist. For buttons we'd have to add that variable to existing margin everywhere buttons are or may be used and I didn't like that.

The other thing to consider though is that if we want to prevent the animation from affecting the shadow on inputs (which I assume we do) we'll need to change our approach to input border anyway.